gtsfm/bundle/bundle_adjustment.py

```diff
@@ -61,12 +63,56 @@
 logger = logger_utils.get_logger()
 
 
-class BundleAdjustmentOptimizer:
+class TwoViewBundleAdjustmentProcess(GTSFMProcess):
+    """Dummy class to get the UI graph correct.
```

Comment:
Let's refine this language a bit -- maybe something like
"""Dummy class that exists in order to visualize the 2-view BA process in a process graph visualization."""